database/dbseeding/mongoModels.js

```diff
@@ -0,0 +1,67 @@
+const mongoose = require('mongoose');
+const db = require('../mongoConnect.js');
+
+const { Schema } = mongoose;
+
+const restaurantsSchema = new Schema({
+  restaurantName: { type: String, required: true },
+  popularDishes: [{
+    type: Schema.Types.ObjectId,
+    ref: 'dishes',
+  }],
+});
+
+const dishesSchema = new Schema({
+  dishName: { type: String, required: true },
+  price: Number,
+  description: String,
+  restaurantId: {
+    type: Schema.Types.ObjectId,
+    ref: 'restaurants',
+    required: true,
+  },
+  reviews: [{
+    type: Schema.Types.ObjectId,
+    ref: 'reviews',
+  }],
+});
+
+const reviewsSchema = new Schema({
+  dishId: {
+    type: Schema.Types.ObjectId,
+    ref: 'dishes',
+    required: true,
+  },
+  userId: {
+    type: Schema.Types.ObjectId,
+    ref: 'users',
+    required: true,
+  },
+  photoUrl: { type: String, required: true },
+  caption: String,
+  createdAt: Number,
+  updatedAt: Number,
+  rating: { type: Number, required: true },
+  reviewText: String,
+});
+
+// const photosSchema = new Schema({

```

Comment:
Do you need this anymore? You've included your photo data in your reviewsSchema, right?

Reply:
I will flag this for removal. Thanks